deploy/osd-delete-backplane-serviceaccounts/10-delete-backplane-serviceaccounts.rbac.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: osd-delete-backplane-serviceaccounts
+  namespace: openshift-backplane
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: osd-delete-serviceaccounts
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+    verbs:
+      - get
+      - list
+      - delete
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: osd-get-namespace
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - namespaces
+    verbs:
+      - get
+      - list
+---
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: osd-delete-backplane-serviceaccounts
+  namespace: openshift-rbac-permissions
+spec:
+  clusterPermissions:
+    - osd-get-namespace
+  permissions:
+    - allowFirst: true
+      clusterRoleName: osd-delete-serviceaccounts
+      namespacesAllowedRegex: "(^openshift-backplane-.*)"
+  subjectKind: User
```

Comment:
```suggestion
  subjectKind: ServiceAccount
```

Reply:
rbac operator doesn't currently handle type of `ServiceAccount`
https://github.com/openshift/rbac-permissions-operator/blob/f0cfa43269218d756876792834ecd6c1209d8b01/pkg/controller/utils/controllerutil.go#L105-L109

The rolebinding definition for a sa is:
```yaml
subjects:
- kind: ServiceAccount
  name: osd-delete-backplane-serviceaccounts
  namespace: openshift-backplane
```

Reply:
@jewzaam This is the last issue on this PR. Seem ok?

Reply:
Yea, if User works then OK.  If not we'll know in stage.